core/primitives/src/trie_key.rs

```diff
@@ -50,8 +48,15 @@ pub mod col {
     /// This column id is used when storing the postponed PromiseYield receipts
     /// (`primitives::receipt::Receipt`).
     pub const PROMISE_YIELD_RECEIPT: u8 = 12;
-    /// All columns except those used for the delayed receipts queue and the yielded promises
-    /// queue, which are both global state for the shard.
+    /// Indices of outgoing receipts. A singleton per shard.
+    /// (`primitives::receipt::BufferedReceiptIndices)
+    pub const BUFFERED_RECEIPT_INDICES: u8 = 13;
+    /// Outgoing receipts that need to be buffered due to congestion +
+    /// backpressure on the receiving shard.
+    /// (`primitives::receipt::Receipt`).
+    pub const BUFFERED_RECEIPT: u8 = 14;
+    /// All columns except those used for the delayed receipts queue, the yielded promises
+    /// queue, and the outgoing receipts buffer, which are global state for the shard.
```

Comment:
Can you add a small comment for the next person adding columns that we are out of nibble size numbers and that 15 needs to be a demux?